temboardui/model/alembic/versions/application.sql

```diff
@@ -0,0 +1,82 @@
+CREATE SCHEMA application;
```

Comment:
Where does this new sql script comes from? Is it copied from elsewhere?

Reply:
C'est déplacé depuis `share/sql`, en retirant le `DROP IF EXISTS` préliminaire.

Reply:
J'ai découplé l'ajout de la version alembic et la suppression du fichier manuel pour faire une bascule d'une méthode à l'autre dans `create_repository.sh` dans un commit dédié.